nilearn/signal.py

```diff
@@ -417,6 +417,13 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         nilearn.image.clean_img
     """
 
+    if isinstance(low_pass, bool):
+        raise TypeError("low pass must be float or None but you provided "
+                        "low_pass='{0}'".format(low_pass))
```

Comment:
error message for wrong input has single quotes which I don't think it is necessary. May be we can remove single quotes ? What do you think ?

```python
TypeError: low pass must be float or None but you provided low_pass='False'
```